psignifit/sigmoids.py

```diff
@@ -144,73 +139,6 @@ def _slope(self, stimulus_level: np.ndarray, threshold: np.ndarray, width: np.nd
     def _inverse(self, prop_correct: np.ndarray, threshold: np.ndarray, width: np.ndarray) -> np.ndarray:
         raise NotImplementedError("This should be overwritten by an implementation.")
 
-    def assert_sanity_checks(self, n_samples: int, threshold: float, width: float):
```

Comment:
I agree with moving this to the tests, but I think the purpose of this method was also to help the implementer of a custom sigmoid, i.e. a user. So you would write your own sigmoig by inheriting from this class and then run the superclass sanity checks to make sure that you indeed wrote something which looks like a sigmoid.

By moving the method to the tests we are hiding this from the user.

Reply:
moved to own function in same module